source/documentation/adrs/adr-021.html.md.erb

```diff
@@ -0,0 +1,34 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: ADR-021 Management of DNS Records through OctoDNS
+last_reviewed_on: 2024-06-25
+review_in: 6 months
+---
+
+# ADR-021 Managing DNS Records Through OctoDNS
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The Operations Engineering (OE) team at the Ministry of Justice (MoJ) recognized the need for a system to manage DNS records consistently across multiple providers and teams. Currently, DNS management is performed leading to inconsistencies and a lack of traceability.
+We require a solution that supports automation, provides a unified configuration format, and integrates seamlessly with our existing CI/CD pipelines.
+
+## Decision
```

Comment:
This should be based on the finding from the PoCs and should also cover the Terraform PoC. The decision should cover why we chose one over the other. See related issues:

[Use OctoDNS to Manage Estate](https://github.com/ministryofjustice/operations-engineering/issues/4536)
[Split Terraform State by Hosted Zone](https://github.com/ministryofjustice/operations-engineering/issues/4535)